deployment/mesh-infra/storage/kustomization.yaml

```diff
@@ -3,5 +3,7 @@ kind: Kustomization
 
 resources:
 # - directpv/
-# - minio/
-- pvc/
\ No newline at end of file
+- minio/
+# - pv/
```

Comment:
Note down that `pv` isn't meant to be included at the mo. The actual volume provisioning process is machine-dependent and manual for now. But you can use the config for the volumes on the two machines we've built (`devm` and `tv-teadal`) as a starting point.

Reply:
done